### ISSUE TITLE
Chart widget - Bad route generated by artisan

### ISSUE THREAD

**Issue by dividy** [Laravel-Backpack/CRUD#2955]
# Bug report

The artisan command used to create a chart widget generates a bad route.

### What I did
```zsh
php artisan backpack:chart SitesPerStatus
```

### What I expected to happen
```php
    Route::get('charts/sites-per-status', 'Charts\SitesPerStatusChartController@response')->name('charts.sites-per-status.index');
```

### What happened
```php
    Route::get('charts/sites-per-status', 'Charts\SitesPerStatusChartController@response')->name('charts.'.sites-per-status.'.index');
```

### What I've already tried to fix it

I change it manually when needed ;)

### Backpack, Laravel, PHP, DB version

When I run ```php artisan backpack:version``` the output is:
```zsh
### PHP VERSION:
PHP 7.2.19 (cli) (built: May 29 2019 13:58:59) ( ZTS MSVC15 (Visual C++ 2017) x64 )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.2.0, Copyright (c) 1998-2018 Zend Technologies

### LARAVEL VERSION:
v7.13.0@6fa69bfbd57744a5bbec5538ce483919b3fd625f

### BACKPACK VERSION:
4.1.6@cbd4143d3eb8302916012af205565cd3183f274f
```

**Comment by iMokhles** [comment]
it's already fixed here https://github.com/Laravel-Backpack/Generators/pull/86

**Comment by tabacitu** [comment]
Oups! Thanks @dividy and @iMokhles , just merged #86 and tagged it! A `composer update backpack/generators` should do it for you guys. Thanks again, cheers!
